week_3/2018202113HYH/src/tools.h

```diff
@@ -0,0 +1,137 @@
+#ifndef TOOLS_H
+#define TOOLS_H
+
+#include <algorithm>
+#include <vector>
+#include <regex>
+#include <queue>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <set>
+#include <ctime>
+
+class get
```

Comment:
It's better to name a class with a *noun*, not a *verb*. e.g. `getter` here instead.